projects/core/src/datetimepicker/calendar.scss

```diff
@@ -1,20 +1,16 @@
 @import '~@angular/material/theming';
-
```

Comment:
just a hint for future contributions.. to any project..
such changes (reformattings, removal of line breaks, etc..) are often not very liked to see.
* formatting was done for a reason
* it makes reviewing PRs hard
* it's kind of a "i dont like your style guides" (which is.. well, not quit friendly)
I can overlook at this time.. but, if you find the time: please undo it ;)

Reply:
Sorry about that, it was not my intention, my vscode was configured to format on save, but I reformat the file following your style guide.